task/testmap.py

```diff
@@ -68,7 +68,7 @@
     'cockpit-project/starter-kit': {
         'master': [
             'centos-7',
-            'fedora-30',
+            'fedora-31',
             'centos-8-stream',
         ],
         '_manual': [
```

Comment:
It is in manual, so it should be tested here and also dropped from manual.

Reply:
Argh, sorry. Fixed/done.